lib/Controller/PersonController.php

```diff
@@ -172,6 +171,49 @@ public function find(int $id) {
 		return new DataResponse($resp);
 	}
 
+	/**
+	 * @NoAdminRequired
+	 */
+	public function findSimilar(int $id) {
+		$deviation = $this->settingsService->getDeviation();
+
+		$enabled = (version_compare(phpversion('pdlib'), '1.0.2', '>=') && ($deviation > 0.0));
+
+		$resp = array();
+		$resp['enabled'] = $enabled;
+		$resp['suggestions'] = array();
+
+		if (!$enabled)
+			return new DataResponse($resp);
+
+		$sensitivity = $this->settingsService->getSensitivity();
+		$modelId = $this->settingsService->getCurrentFaceModel();
+
+		$mainPerson = $this->personMapper->find($this->userId, $id);
+		$mainFace = $this->faceMapper->findRepresentativeFromPerson($this->userId, $id, $sensitivity, $modelId);
+
+		$suggestions = array();
+		$persons = $this->personMapper->findAll($this->userId, $modelId);
```

Comment:
I didn't test, but I hardly think this will not timeout for me. I have hundreds of persons with thousands of faces, this will hit DB very hard:D

Reply:
This was the comment I expected. haha..
...and for this made it configurable and disabled by default ..

Reply:
haha, I am *that* predictable?:D